openlibrary/catalog/edition_merge/templates/index.html

```diff
@@ -1,61 +1,69 @@
 <html>
+
 <head>
-<title>Merge duplicate editions</title>
```

Comment:
These whitespace changes seem unrelated and make this pull request harder to read and understand. Can you remove?

Reply:
DONE